controlpanel/frontend/jinja2/user-list.html

```diff
@@ -29,20 +29,17 @@ <h1 class="govuk-heading-xl">{{ page_title }}</h1>
       </td>
       <td class="govuk-table__cell">{{ user.email }}</td>
       <td class="govuk-table__cell">
-        {%- with login_details=last_login.get(user.auth0_id) -%}
-          {%- if login_details -%}
-          {%- with login = login_details.get("last_login") -%}
-          <span data-last-login="{{login}}"
-                title="{{login.strftime("%Y/%m/%d %H:%M:%S")}}">
-            {{ timesince(login) }} ago
-          </span>
-          {%- if login_details.get("unused") -%}
-          (<strong><a href="https://github.com/orgs/moj-analytical-services/people/{{ user.username }}" target="_blank">unused?</a></strong>)
-          {%- endif -%}
-          {%- endwith -%}
+          {%- if user.last_login -%}
+              <span data-last-login="{{user.last_login}}"
+                    title="{{user.last_login.strftime("%Y/%m/%d %H:%M:%S")}}">
+                {{ timesince(user.last_login) }} ago
+              </span>
           {%- else -%}
+              <span>Never logged in.</span>
```

Comment:
I wonder if it's worth having the link in this branch as well or not. Although I don't imagine many users will not have a this blank.

Reply:
There is a link. If they've not logged in they're marked as un-used users.